petasos-rewriter.yaml

```diff
@@ -85,4 +85,9 @@ metricsOptions:
   namespace: xmidt
   # subsystem is the subsystem of the metrics provided
   # (Optional)
-  subsystem: petasos_rewriter
\ No newline at end of file
+  subsystem: petasos_rewriter
+
+# If enabled, requests redirected to Talaria will be examined for the presence of an Authorization header, 
+# and if the Authorization header is not present, the request will be rejected.
+authHeaderCheck:
```

Comment:
make it like  server.authheader.check.enabled.
 server.authheader.check.requestPath:

Reply:
Done.